src/EdFi.Admin.LearningStandards.Core/Services/FromCsv/DataMappingProcess.cs

```diff
@@ -0,0 +1,175 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using EdFi.Admin.LearningStandards.Core.Models.FromCsv;
+using EdFi.Admin.LearningStandards.Core.Services.Interfaces.FromCsv;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace EdFi.Admin.LearningStandards.Core.Services.FromCsv
+{
+    public class DataMappingProcess : IDataMappingProcess
+    {
+        public string DataMappingsFilePath { get; set; } 
+
+        public JObject ApplyMap(IEnumerable<LearningStandardMetaData> learningStandardMetaData,
+            IEnumerable<DataMapper> dataMappers, Dictionary<string, string> csvRow)
+        {
+            var safeCsvRow = new CsvRow(csvRow);
+
+            return MapToJsonObject(learningStandardMetaData.ToList(), dataMappers.ToList(), safeCsvRow);
+        }
+
+        public IEnumerable<DataMapper> GetDataMappings()
+        {
+            if (string.IsNullOrEmpty(DataMappingsFilePath))
+            {
+                DataMappingsFilePath = ".//Mappings.json";
```

Comment:
I would store the string value as a private field, incase this needs to become configurable in the future (through the constructor, method, etc.).